### PR TITLE
thinktandem/horoscope#182: Blog Post: Lando and Heroku friends forever.

### PR DESCRIPTION
**Pull request by serundeputy**

### CODE REVIEW

File: src/content/articles/lando-heroku-workflow.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+byline: "🚀 ∞ Lando + Heroku: take your dev to the next level.  📦 ☄️"
+date: 2017-12-01
+---
+
+Lando + Heroku Workflow
+-----------------------
+
+[Heroku](https://heroku.com) is a flexible cloud hosting platform. [Lando](https://docs.devwithlando.io) is a flexible local development environment. Together your dev workflow can really shine!
+
+In and of itself there is no reason you can't just take any app and deploy to Heroku as you normally would; this post will walk you through constructing a useful `.lando.yml` config file for working with Heroku.
```
Comment (Senpai) [inline src/content/articles/lando-heroku-workflow.md]: "In and of itself," is a clause, and need a trailing comma.

File: src/content/articles/lando-heroku-workflow.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+Heroku CLI
+----------
+
+Heroku has a powerful cli tool to interact with your Heroku hosted apps. Let's take advantage of that and integrate it into lando. Start a `.lando.yml` file like this:
```
Comment (Senpai) [inline src/content/articles/lando-heroku-workflow.md]: Capitalize "Lando".

File: src/content/articles/lando-heroku-workflow.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+```
+
+The `appserver extras` section:
+
+```yaml
+    extras:
+      - "wget -qO- https://cli-assets.heroku.com/install-ubuntu.sh | sh"
+```
+
+goes out and gets the `heroku` cli app and installs in the `appserver` container.
```
Comment (Senpai) [inline src/content/articles/lando-heroku-workflow.md]: "goes out and gets" is a colloquialism, and might be better described with a word like "retrieves"?

File: src/content/articles/lando-heroku-workflow.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+The `appserver extras` section:
+
+```yaml
+    extras:
+      - "wget -qO- https://cli-assets.heroku.com/install-ubuntu.sh | sh"
+```
+
+goes out and gets the `heroku` cli app and installs in the `appserver` container.  
+
+The `tooling heroku` section:
```
Comment (Senpai) [inline src/content/articles/lando-heroku-workflow.md]: Maybe indicate a hierarchical relationship between these two words with a "tooling > heroku" or a "tooling: heroku" type structure?

File: src/content/articles/lando-heroku-workflow.md [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+
+```yaml
+heroku:
+  service: appserver
+  cmd: /usr/bin/heroku
+```
+
+Adds it as an available command to your lando app; so now you can run `lando heroku`:
+
+Now start your app and now you have access to the `heroku` cli.
```
Comment (Senpai) [inline src/content/articles/lando-heroku-workflow.md]: "...now you have access to the `heroku` cli" You also state this fact on line 135. Recommend getting rid of one or the other.

File: src/content/articles/lando-heroku-workflow.md [hunk 6]
```diff
@@ -0,0 +1,203 @@
...
+lando heroku auth:login
+```
+
+Create an app on Heroku
+-----------------------
+
+For this example we'll build a [Lavarvel](https://laravel.com) app:
+
+```bash
+lando heroku apps:create lando-heroku --buildpack heroku/php
```
Comment (Senpai) [inline src/content/articles/lando-heroku-workflow.md]: This is dang sexy!

File: src/content/articles/lando-heroku-workflow.md [hunk 7]
```diff
@@ -0,0 +1,203 @@
...
+```
+
+Get info about your Heroku app with the `apps:info` command:
+
+```bash
+geoff@yep lando-heroku $ lando heroku apps:info lando-heroku
```
Comment (Senpai) [inline src/content/articles/lando-heroku-workflow.md]: `geoff@yep` For a public instructional article, sanitize the personal machine name information in order to make it more relate-able to the masses.

File: src/content/articles/lando-heroku-workflow.md [hunk 8]
```diff
@@ -0,0 +1,203 @@
...
+
+Conclusion
+----------
+With a few simple configuration files and commands you are ready to go with all your dev tools and a chain from local to live. You can see the full git repo on GitHub: https://github.com/thinktandem/lando-heroku-demo. The main files of importance are:
+
+* [`Procfile`](https://github.com/thinktandem/lando-heroku-demo/blob/master/Procfile)
+* [`.lando.yml`](https://github.com/thinktandem/lando-heroku-demo/blob/master/.lando.yml)
+* [`nginx_app.conf`](https://github.com/thinktandem/lando-heroku-demo/blob/master/nginx_app.conf)
+
+🐼 All the convenience of Lando all the flexibility of Heroku; enjoy! 🐝
```
Comment (Senpai) [inline src/content/articles/lando-heroku-workflow.md]: "All the convenience of Lando, all the flexibility of Heroku"

Comment (serundeputy) [reply src/content/articles/lando-heroku-workflow.md]: Thanks @Senpai !

Great feedback; I've incorporated your suggestions and updated the PR.

**Pull request merged**